wie-geht-studium/mafin.tex

```diff
@@ -0,0 +1,12 @@
+\subsection{Mathematik für Informatiker}
+\label{mafin}
+
+Die Mathematikausbildung für InformatikerInnen ist etwas kompliziert. Ihr habt die Auswahl zwischen den Vorlesungen Lineare Algebra I und Analysis I oder den Vorlesungen Mathematik für Informatiker I und II. Ihr entscheidet euch also bereits im ersten Semester für eine der beiden Varianten. Deshalb solltet ihr euch schon jetzt mit diesem Thema auseinandersetzen.\\
+
+Was spricht für Mathematik für Informatiker (MafIn)?\\
+Die Vorlesung ist extra auf InformatikerInnen zugeschnitten und versucht euch den Stoff besser verständlich näher zu bringen. Die Mathematik für Informatiker I orientiert sich dabei an den grundlegenden Inhalten der Linearen Algebra I, die Mathematik für Informatiker II an denen der Analysis I. Grundsätzlich ist die Vorlesung recht solide konzipiert und kann euch die wichtigen Mathematik-Grundlagen vermitteln. Leider wurde diese Vorlesung in den letzten Jahren immer vom selben Dozenten gehalten, mit dem es einige Probleme in der Vorlesung und dem zugehörigen Übungsbetrieb gab. Ihr solltet euch aber selbst ein Bild von der Vorlesung machen, ob sie euch zusagt oder nicht. Lasst uns gerne Feedback zur Vorlesung zu kommen\footnote{Feedback könnt ihr uns am besten per Mail, oder bei der Informatik-Vollversammlung zukommen lassen.}.\\
+
+Was spricht für Lineare Algebra I (LA I)\footnote{Details zur LA I siehe Seite \pageref{la1}} und Analysis I (Ana I)\footnote{Details zur Ana I siehe Seite \pageref{ana1}}?\\
+Die Vorlesungen sind für MathematikerInnen konzipierte Veranstaltungen. Diese bieten dadurch eine deutlich präzisere Herangehensweise an und Formulierung der Definitionen, Sätze und insbesondere Beweise. Ihr bekommt dadurch eine deutlich solidere Mathematikausbildung die ihr in manchen Bereichen der Informatik auch dringend benötigt. Die LA I und Ana I bedeutet für euch aber auch meist mehr Arbeit und Zeitaufwand, da ihr auch eine deutlich intensivere und vertiefendere Ausbildung bekommt.\\
+
+Schlussendlich muss jeder selbst wissen was für sein Studium und sein Studienplan das beste ist. Informiert euch möglichst zu Beginn eures Studiums was für euch besser passt. Im Zweifel könnt ihr euch auch in beide Vorlesungen rein setzen und die Art der Vorlesungen vergleichen. Macht dies aber nicht länger als ein paar Wochen, da der Mehraufwand gerade im ersten Semester sehr hoch ist. Im Zweifel bekommt ihr in der Regel während des Vorkurses auch noch einen Vortrag zum Studienaufbau bei dem ihr noch Fragen stellen könnt. Ansonsten freuen wir uns wenn ihr mit Fragen und Anregungen zu dieser schwierigen zu uns\footnote{Mehr Infos zur Fachschaft im \autoref{diefsmathphys}.} kommt
```

Comment:
Letzter Satz. Sie sollen mit dieser  "schwierigen" was zu uns kommen? Felt also noch ein Wort in Richtung Auswahl oder Problemstellung.